aws/sdk/integration-tests/codecatalyst/tests/sso_bearer_auth.json

```diff
@@ -0,0 +1,126 @@
+            "action": {
+                "Data": {
+                    "data": {
+                        "Utf8": "{}"
+                    },
+                    "direction": "Request"
+                }
+            }
+        },
+        {
+            "connection_id": 0,
+            "action": {
+                "Eof": {
+                    "ok": true,
+                    "direction": "Request"
+                }
+            }
+        },
+        {
+            "connection_id": 0,
+            "action": {
+                "Response": {
+                    "response": {
+                        "Ok": {
+                            "status": 200,
+                            "headers": {
+                                "x-amzn-served-from": [
+                                    "us-west-2,us-west-2"
+                                ],
+                                "access-control-allow-credentials": [
+                                    "true"
+                                ],
+                                "content-type": [
+                                    "application/json; charset=utf-8"
+                                ],
+                                "strict-transport-security": [
+                                    "max-age=63072000; includeSubDomains"
+                                ],
+                                "access-control-expose-headers": [
+                                    "anti-csrftoken-a2z,x-amzn-requestid"
+                                ],
+                                "date": [
+                                    "Tue, 27 Feb 2024 00:12:28 GMT"
+                                ],
+                                "vary": [
+                                    "Origin"
+                                ],
+                                "x-amzn-requestid": [
+                                    "some-request-id"
+                                ]
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        {
+            "connection_id": 0,
+            "action": {
+                "Data": {
+                    "data": {
+                        "Utf8": "{\"nextToken\":\"\",\"items\":[{\"name\":\"somespacename\",\"regionName\":\"us-west-2\",\"displayName\":\"somedisplayname\"}]}"
+                    },
+                    "direction": "Response"
+                }
+            }
+        },
+        {
+            "connection_id": 0,
+            "action": {
+                "Data": {
+                    "data": {
+                        "Utf8": ""
+                    },
+                    "direction": "Response"
+                }
+            }
+        },
+        {
+            "connection_id": 0,
+            "action": {
+                "Eof": {
+                    "ok": true,
+                    "direction": "Response"
+                }
+            }
+        }
+    ],
+    "docs": "todo docs",
```

Comment:
It'd be good to edit `todo docs`.

Reply:
Fixed in 89a2ea8895f32066e1ad803ec634683c307b0c84